### ISSUE TITLE
Do everything in your power to reduce the file size of the new Wifi image

### ISSUE THREAD

**Issue by george08** [museuminabox/brain#46]
8GB seems like a lot! (And, would also mean we'll have to get 16GB SD cards as a minimum in future, so BOM would have to change.)

**Comment by amcewen** [comment]
Does writing to an 8GB card not work?  The expanded img files end up that big because that's how big the SD cards I'm using are - they're not 8GB of /used/ space, just 8GB because it's a copy of a the entire card.

**Comment by george08** [comment]
@CharlieMiab, your attention, please!

**Comment by CharlieMiab** [comment]
No, Etcher refuses to let a 8GB card be written to even if the card has been wiped because the unzipped file still shows up as 8.02GB

**Comment by amcewen** [comment]
We worked out that this was because not all SD cards are created equal :-/

Specifically, some 8GB cards are slightly bigger than others, and so if I create an img file from a bigger one, it can't be written to the smaller ones.  So, I need to make sure I build the "blank base img" files on the smaller cards.

**Comment by Rjevski** [comment]
This is becoming an issue again as we need to write a lot of images and writing the full 8GB (most of which is free space) is wasteful.

I've looked at https://github.com/Drewsif/PiShrink and successfully ran it in an Ubuntu 18.04.3 VM; however I have not had the opportunity to test the new image to see if it resizes itself back to the proper size on first boot.

@CharlieMiab has the resulting image and is going to test tomorrow as per my instructions (etch the image, boot a box, SSH into it and run `df -h` to see if the partition is the proper size).
